pkg/operator/ceph/controller/spec.go

```diff
@@ -934,3 +936,22 @@ func GenerateLivenessProbeViaRpcinfo(port uint16, failureThreshold int32) *v1.Pr
 		FailureThreshold:    failureThreshold,
 	}
 }
+
+func SkipReconcile(ctx context.Context, clusterd *clusterd.Context, namespace, daemonName, label string) (sets.Set[string], error) {
```

Comment:
How about this name?
```suggestion
func GetDaemonsToSkipReconcile(ctx context.Context, clusterd *clusterd.Context, namespace, daemonName, label string) (sets.Set[string], error) {
```

Reply:
done